components/Relationships/RelationshipElement.spec.tsx

```diff
@@ -32,4 +33,14 @@ describe('Relationships component', () => {
     expect(getByText('Unborn sibling')).toBeInTheDocument();
     expect(getByText('Jambi Neverborn')).toBeInTheDocument();
   });
+  it('should populate the list in alphabetical order', async () => {
+    const { queryAllByText } = render(
+      <RelationshipElement {...mockedOrderedRelationship} />
```

Comment:
Shouldn't the relationships we provide be unordered so we know that the code then orders it?

When we say alphabetical order, do we mean by first name or last name? Maybe we can update the test description to be a bit more explicit.

Could we also provide another test example where the names don't have the same surname?

Reply:
Fair enough :D

I've now 
-mixed the people's order (so they're not in order in the tests)
-added different name/surname in the list
-added a proper description to the test